e2e/specs/register-new-patient.spec.ts

```diff
@@ -25,23 +24,58 @@ test('Register a new patient', async ({ page }) => {
     stateProvince: 'Pernambuco',
     cityVillage: 'Recife',
     phone: '5555551234',
+    email: 'johnnyronny@example.com',
   };
 
   await test.step('When I visit the registration page', async () => {
     await patientRegistrationPage.goto();
     await patientRegistrationPage.waitUntilTheFormIsLoaded();
   });
 
-  await test.step('And then I fill the registration form with the data in `formValues` and then click the `Submit` button', async () => {
-    // Check for explicit visibility of address template fields
-    await expect(patientRegistrationPage.addressHierarchySearchInput()).toBeVisible();
-    await expect(patientRegistrationPage.address1Input()).toBeVisible();
-    await expect(patientRegistrationPage.countryInput()).toBeVisible();
-    await expect(patientRegistrationPage.stateProvinceInput()).toBeVisible();
-    await expect(patientRegistrationPage.cityVillageInput()).toBeVisible();
```

Comment:
These steps to check if the address fields are visible should not be removed. Please add it back either at the start of the test or to the step where these relevant fields are filled in (preferably the latter)